scripts/after-pack-hook.js

```diff
@@ -10,67 +10,74 @@ const { buildEntryPointAndCleanup } = require('./binary/binary-cleanup')
 const { getIntegrityCheckSource, getBinaryEntryPointSource } = require('./binary/binary-sources')
 
 module.exports = async function (params) {
-  console.log('****************************')
-  console.log('After pack hook')
-  console.log(params.appOutDir)
-  console.log(params.outDir)
-  console.log(params.electronPlatformName)
-  console.log('****************************')
-
-  const packages = glob.sync('packages/*/node_modules', {
-    cwd: params.packager.info._appDir,
-  })
-
-  const buildSubfoldersPerPlatform = {
-    darwin: join('Cypress.app', 'Contents', 'Resources', 'app'),
-    linux: join('resources', 'app'),
-    win32: join('resources', 'app'), // TODO check this path
-  }
-  const buildSubfolder = buildSubfoldersPerPlatform[os.platform()]
-  const outputFolder = join(params.appOutDir, buildSubfolder)
+  try {
```

Comment:
I wouldn't have expected test changes to affect build scripts, why are these changes needed?

Reply:
Oops forgot to comment on the description for why this was changed. I added a couple of changes to be able to move `bytenode` from a main dependency to a dev dependency and just generally cleaned some of this up. No functional changes, but it was something that was nagging at me from my last PR that I wanted to clean up while I had time.

Reply:
Added this to the description.